### PR TITLE
ci: add check for expected number of elders after startup

### PR DESCRIPTION
**Pull request by joshuef**

### CODE REVIEW

**Comment by joshuef** [comment]
bors try

File: sn_comms/src/lib.rs [hunk 1]
```diff
@@ -190,49 +190,28 @@ impl Comm {
...
-                    } else {
-                        if should_remove {
-                            // do cleanup of that peer
-                            let perhaps_session = sessions.remove(&peer);
-                            if let Some((_peer, session)) = perhaps_session {
-                                session.disconnect().await;
-                            }
-                        }
-                        Err(Error::FailedSend(peer))
+            let (send_was_successful, should_remove) = Self::is_sent(watcher, msg_id, peer).await?;
```
Comment (RolandSherwin) [inline sn_comms/src/lib.rs:201]: Not directly related, but the `Result` in the `is_sent` method is redundant and can be removed..

Comment (joshuef) [reply sn_comms/src/lib.rs]: sorted

**Review by RolandSherwin — APPROVED** [review]
The changes look good to me!

**Pull request merged**